documentation/api/command/v1/commands.markdown

```diff
@@ -166,6 +180,14 @@ To post a `replace facts` command you can use the following curl command:
 An example of `deactivate node`:
 
     curl -X POST \
+      -H 'Content-Type:application/json' \
+      -H 'Accept:application/json' \
+      -d '{"certname":"test1","producer_timestamp":"2015-01-01"}' \
```

Comment:
I think we should avoid this for now. We'll end up having to write that it's optional in one case but not the other, which starts to feel like a separate version. I'd vote for leaving it until we can bump the command and retire the old mechanism.